build.sbt

```diff
@@ -27,14 +26,13 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "2.2.6" % Test
 )
 
-
 dependencyOverrides ++= Set(
   "org.apache.httpcomponents" % "httpclient" % "4.5.1",
   "org.slf4j"                 % "slf4j-api"  % "1.7.7"
 )
 
 
-fatArtifactSettings
+addFatArtifactPublishing()
```

Comment:
by the way, the argument of this method is configuration. It's `Compile` by default, but you can try to use it for `Test`